pkg/csi/cinder/identityserver.go

```diff
@@ -46,6 +47,11 @@ func (ids *identityServer) GetPluginInfo(ctx context.Context, req *csi.GetPlugin
 }
 
 func (ids *identityServer) Probe(ctx context.Context, req *csi.ProbeRequest) (*csi.ProbeResponse, error) {
+	_, err := openstack.CreateOpenStackProvider()
+	if err != nil {
+		klog.V(3).Infof("Failed to CreateOpenStackProvider: %v", err)
```

Comment:
Shouldn't the log level be ERROR?

Reply:
right, but actually I copied the code :)
we may need another PR to fix those..

Reply:
ok, good to know, thanks!